src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo | null,
+  onCloseModal: () => void,
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, onCloseModal }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const loadUser = async () => {
+    try {
+      const loadedUser = await getUser(todo?.userId || 0);
+
+      setUser(loadedUser);
+    } catch (error) {
+      throw new Error(`There is an error with loading data: ${error}`);
```

Comment:
same here about errors processing, let's think how we can handle it without throwing new error